itemspawner/sh_plugin.lua

```diff
@@ -0,0 +1,27 @@
+local PLUGIN = PLUGIN
+
+PLUGIN.name = "Item Spawner System"
+PLUGIN.author = "Gary Tate"
+PLUGIN.description = "Allows staff to select item spawn points with great configuration."
+
+PLUGIN.spawner = PLUGIN.spawner or {}
+PLUGIN.items = PLUGIN.items or {}
+PLUGIN.spawner.positions = PLUGIN.spawner.positions or {}
+
+CAMI.RegisterPrivilege({
```

Comment:
Helix creates the privilege specified in a command with the `Helix - ` prefix if it doesn't exist. You should be able to remove this and have it work without having to rename the privileges